src/main/webapp/busmap.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Bus Stops in Barcelona</title>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBhWNDhXcbviYWwpu1Ksf5u0fZnHpW5c3E"></script>
+  <script>  
+    function createBusStopsMap(){
+      fetch('/bus-stops').then(function(response) {
+        return response.json();
+      }).then((busStops) => {
+        const map = new google.maps.Map(document.getElementById('map'), {
+          center: {lat: 41.3896, lng: 2.1624},
+          zoom:12
+        });
+
+        busStops.forEach((busStop) => {
+          var marker = new google.maps.Marker({
+            position: {lat: busStop.lat, lng: busStop.lng},
+            map: map,
+            title: busStop.neighborhood
```

Comment:
this should be label, not title

Reply:
But label is modifying pin itself. title should have the text we want next to the pin.